tools/spelling/check_spelling.sh

```diff
@@ -71,12 +71,12 @@ chmod +x "${TMP_DIR}/misspell"
 # Spell checking
 # All the skipping files are defined in tools/spelling/spelling_skip_files.txt
 SPELLING_SKIP_FILES="${ROOTDIR}/tools/spelling/spelling_skip_files.txt"
+IFS=' ' read -ra SKIP_FILES < "$SPELLING_SKIP_FILES"
+IFS=' ' read -ra SKIP_FILES <<< "${SKIP_FILES[@]/#/-e }"
```

Comment:
@zuercher i think this is the most likely candidate for backwards incompatibility

Reply:
this works on `3.1`:

```
$ docker run bash:3.1 -c 'export FOO=(X Y Z); echo ${FOO[@]/#/A}'
AX AY AZ
```

but fails on anything less.

afaict that means it works on osx (at least recent versions)